pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -118,10 +118,31 @@ type BundleTarget struct {
 type AdditionalFormats struct {
 	// JKS requests a JKS-formatted binary trust bundle to be written to the target.
 	// The bundle is created with the hardcoded password "changeit".
-	JKS *KeySelector `json:"jks,omitempty"`
+	JKS *JKS `json:"jks,omitempty"`
 	// PKCS12 requests a PKCS12-formatted binary trust bundle to be written to the target.
 	// The bundle is created without a password.
```

Comment:
suggestion: Looks like this comment will need updating. Might be worth linking to https://cert-manager.io/docs/faq/#keystore-passwords as well!